Base/ConnectorValue.cs

```diff
@@ -21,7 +21,7 @@ public override string ToString()
                     result = String;
                     break;
                 case FSUIPCOffsetType.Integer:
-                    result = Float64.ToString("R");
+                    result = Math.Round(Float64).ToString();
```

Comment:
Why is this PR doing it this way and [the other PR used "R"?](https://github.com/MobiFlight/MobiFlight-Connector/pull/1182/files)?

Reply:
Connector Value remembers what the original value type was. Internally it uses always floats.

At the moment, Mobiflight converts the ConnectorValue using ToString() before sending it to a display.

ToString() now converts the number to a whole number (round) when it originally was a FSUIpCOffsetType.Integer

Reply:
this really only applies to FSUIPC offsets and the weird transformations (eg, heading is $*360/65000) but these values are commonly used in the context of FSUIPC.